src/goto-harness/memory_snapshot_harness_generator.cpp

```diff
@@ -207,16 +207,16 @@ size_t memory_snapshot_harness_generatort::pointer_depth(const typet &t) const
     return pointer_depth(t.subtype()) + 1;
 }
 
-bool memory_snapshot_harness_generatort::refers_to(
+void memory_snapshot_harness_generatort::collect_references(
```

Comment:
While I understand the need for the change for this, I feel slightly uneasy about the change to the return value (which provides some sort of feedback) to a void return value.